packages/smooth_app/lib/helpers/background_task_helper.dart

```diff
@@ -25,6 +25,9 @@ const String NUTRITION_EDIT = 'NUTRITION_EDIT';
 /// Constant for basic details edit task
 const String BASIC_DETAILS = 'BASIC_DETAILS';
 
+/// Constnat for packaging edit task
+const String PACKAGING_EDIT = 'PACKAGING_EDIT';
```

Comment:
What about CATEGORY_EDIT, LABELS_EDIT, STORES_EDIT, ORIGINS_EDIT, PACKAGER_CODES_EDIT, COUNTRY_EDIT ?

Reply:
It's actually handled from `simple_input_page.dart`
`        UniqueIdGenerator.generateUniqueId(changedProduct.barcode!, pageName);
`
I used the `pageNo` variable for that, and checked that, it works
So no problem for these

Reply:
@teolemon is right, @AshAman999 you're cheating: the `pageName` is just the page name of the last item (e.g. of `COUNTRY_EDIT`)
That's why we should refactor the whole thing.

Reply:
I know I won't have time to refactor before Thursday morning.